test/cli/install/registry/bun-install-registry.test.ts

```diff
@@ -5,40 +5,16 @@ import { mkdtempSync, realpathSync } from "fs";
 import { rm, writeFile, mkdir, exists, cp } from "fs/promises";
 import { readdirSorted } from "../dummy.registry";
 import { tmpdir } from "os";
-import { fork, ChildProcess } from "child_process";
 import { beforeAll, afterAll, beforeEach, afterEach, test, expect, describe } from "bun:test";
 
 expect.extend({
   toBeValidBin,
   toHaveBins,
 });
 
-var verdaccioServer: ChildProcess;
 var testCounter: number = 0;
-var port: number = 4873;
 var packageDir: string;
 
-beforeAll(async () => {
```

Comment:
@dylan-conway this won't quite work. We need to check if it's CI and then use the CI version, but in development we still want to do this. I suggest having an `isCI`